### PR TITLE
Add release notes for Bloop 1.4.7

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: notes/v1.4.7.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Installing Bloop
+
+For more details about installing Bloop, please see [Bloop's Installation Guide][install-guide]
+
+## Merged pull requests
+
+Here's a list of pull requests that were merged:
+
+|- Handle jvmOptions without -J prefix in debugSession/start request [#1455]
```
Comment (adpi2) [inline notes/v1.4.7.md:13]: What is this `|` char at the beginning of each line? It doesn't seem to be formatted correctly by github: https://github.com/tgodzik/bloop/blob/release-notes-1.4.7/notes/v1.4.7.md

File: notes/v1.4.7.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+|- fix for autocompletion carriage-return problem in Windows [#1443]
```
Comment (adpi2) [inline notes/v1.4.7.md:14]:
```suggestion
|- Fix for autocompletion carriage-return problem in Windows [#1443]
```

**Review by adpi2 — CHANGES_REQUESTED** [review]

File: notes/v1.4.7.md (hunk 1 above)
Comment (tgodzik) [reply notes/v1.4.7.md:13]: Och no, I trusted my old script and it seems that I still have bug. Good catch.

File: notes/v1.4.7.md (hunk 2 above)
Comment (tgodzik) [reply notes/v1.4.7.md]: Fixed!

File: notes/v1.4.7.md (hunk 1 above)
Comment (tgodzik) [reply notes/v1.4.7.md]: Fixed!

**Comment by tgodzik** [comment]
Btw. the script I used is https://gist.github.com/tgodzik/e4d4bf84db2f18e5cef143acd6730d5f if I forget to add it somewhere sensible.

**Review by adpi2 — APPROVED** [review]

**Pull request merged**